host.js

```diff
@@ -1 +1,128 @@
-module.exports = require('./src/host');
+import { hostResizer } from './plugins/iframe-resizer/host.js';
+import { hostSyncCssVariable } from './plugins/sync-css-variable/host.js';
+import { hostSyncDataAttrs } from './plugins/sync-data-attrs/host.js';
+import { hostSyncFlags } from './plugins/sync-flags/host.js';
+import { hostSyncFont } from './plugins/sync-font/host.js';
+import { hostSyncIntl } from './plugins/sync-intl/host.js';
+import { hostSyncLang } from './plugins/sync-lang/host.js';
+import { hostSyncOslo } from './plugins/sync-oslo/host.js';
+import { hostSyncTimezone } from './plugins/sync-timezone/host.js';
+import { hostSyncTitle } from './plugins/sync-title/host.js';
+import { PortWithServices } from './port/services.js';
+
+const originRe = /^(http:\/\/|https:\/\/)[^/]+/i;
+
+export class Host extends PortWithServices {
+
+	constructor(elementProvider, src, options) {
+		super(undefined, undefined, options);
```

Comment:
I really dislike this, but the way this stuff is structured sets up a chicken/egg problem here. We can't invoke `this` before invoking `super()`, but `super()` takes the `endPoint` and `targetOrigin` as parameters. I'm sure there are ways to refactor this that are better, but I'm also a bit leery about doing more than a relatively straight-forward conversion.

Reply:
Neither of tryGetOrigin or createIframe need to be instance methods. That help?

Reply:
Oh, true, they can be defined outside of the class. Thanks Owen, I've been in the weeds with this for too long. 🤦